smother/control.py

```diff
@@ -41,7 +46,7 @@ def start(self):
 
     def save_context(self, label):
         self.data[label] = {
-            key: sorted(map(int, val.keys()))
+            self._normalize_path(key): sorted(map(int, val.keys()))
```

Comment:
> there is no way (without editing the file ourselves) to get relative paths

You are right -- I just meant you can achieve the "same thing" in that you can create a correct smother file where the paths resolve on the local filesystem, without the flag. The only difference is that the paths themselves are absolute.

I'm conceptually on board with the rest of this PR. If you still feel strongly about `--relative-paths`, can we move that to a dedicated PR and focus this one on implementing path aliasing without the flag?